### DISCUSSION TITLE
How can I implement burst in rate limiting?

### DISCUSSION THREAD

**Discussion by bayungrh** [animir/node-rate-limiter-flexible#273] [Q&A]
I have an external backend API with a rate limit configuration of **20 request per second** and a burst of **10**. How can I use the rate-limiter-flexible library with a similar configuration to limit requests to that API through my application?

Thankyou!

**Comment by animir** [comment]
@bayungrh You can try [BurstyRateLimiter](https://github.com/animir/node-rate-limiter-flexible/wiki/BurstyRateLimiter).
It depends on how the external API does burst allowance.
It is easy to limit per second, but with burst allowance you should understand what 10 means and how quick it is refilled.
Anyway, you could simplify your task to just 20 request per second and then use any limiter from this package.
